environment-non-fsds.yml

```diff
@@ -19,4 +19,5 @@ dependencies:
   - skare3_tools
   - vv_trending
   - twiki-wg
+  - obc_rate_noise_plots
```

Comment:
Newline?

Reply:
this already had two extra lines at the end, so the file still has a newline at the end.